objax/gradient.py

```diff
@@ -60,7 +60,13 @@ def f_func(inputs_and_train_tensors: List[JaxArray],
             finally:
                 self.vc.assign(original_vc)
 
-        self.input_argnums = input_argnums or tuple()
+        if input_argnums is None:
+            self.input_argnums = tuple()
+        else:
+            assert isinstance(input_argnums, tuple), \
+                f"Must pass a tuple of indices to input_argnums; received {input_argnums}."
+            self.input_argnums = input_argnums
```

Comment:
I don't think we need an if statement, we can keep the code simple by changing the assert condition.
```suggestion
        assert isinstance(input_argnums, tuple) or input_argnums is None, \
                f'Must pass a tuple of indices to input_argnums; received {input_argnums}.'
        self.input_argnums = input_argnums or tuple()
```

Also nit, let's use single quotes to keep string consistency with rest of code.